### PR TITLE
raspberrypi: add README containing maintainer information

### PR DESCRIPTION
**Pull request by mirzak**
Per our discussion in #101, @texierp @drewmoseley , here is a suggested template for sub-layers and the information that they should contain (including maintainer)

### CODE REVIEW

**Comment by mirzak** [comment]
@nandra @dwalkes @Valks, feel free to comment as you also have created layers in this layer.

**Comment by nandra** [comment]
:+1:

**Comment by mirzak** [comment]
Updated with a `Dependencies` section, to make the README follow common practices for Yocto/OE-core layers.

**Comment by drewmoseley** [comment]
Looks ok to me.  The only comment I have is do we really want the quick start section?  That gives us two places with the same info which need to be manually kept in sync. Should we just link to the hub post instead?

File: meta-mender-raspberrypi/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# meta-mender-raspberrypi
+
+Mender integration layer for Raspberry Pi family of boards. Note that this layer
+only contains templates, and the actually integration patches live in
```
Comment (dwalkes) [inline meta-mender-raspberrypi/README.md]: Minor nit - I think this should be and the *actual* integration pactches insetad of and the *actually* integration packages.

**Comment by dwalkes** [comment]
@drewmoseley @mirzak +1 for one place to update.  I've had a conversation with @mirzak about this in the past.  My personal preference is to keep more detail with the code rather than the mender hub page to more cleanly handle the cases when documentation changes between releases, however I can see value in both approaches.

File: meta-mender-raspberrypi/README.md (hunk 1 above)
Comment (mirzak) [reply meta-mender-raspberrypi/README.md]: Thanks fixed.

**Pull request merged**